### ISSUE TITLE
Error buffer is focused when it shouldn't be auto-selected

### ISSUE THREAD

**Issue by philc** [clojure-emacs/cider#937]
Steps to reproduce:

1. `(setq cider-auto-select-error-buffer nil)`
2. Eval `(/ 1 0)` in a clojure file buffer (not the cider repl). See the error buffer appear; it does not have keyboard focus.
3. Eval `(/ 1 0)` in the source buffer again. The error buffer is dismissed for a split second and then shown again. Now it has keyboard focus; it shouldn't.

This is in Cider 0.8.2 from MELPA stable. I upgraded Cider recently and this functoinality broke. It worked as expected in a previous version; I believe 0.8.0.

**Comment by cichli** [comment]
@philc this should be fixed on 0.9.0-SNAPSHOT now, if you're willing to give it a test.

**Comment by philc** [comment]
Thanks guys.
